components/esp_common/include/esp_pm.h

```diff
@@ -65,6 +65,11 @@ esp_err_t esp_pm_configure(const void* config);
  */
 typedef struct esp_pm_lock* esp_pm_lock_handle_t;
 
+/**
+ * @brief Automatic light sleep wakeup callback function type
+ */
+typedef void  (*esp_pm_light_sleep_wakeup_callback)();
```

Comment:
it is often useful to pass an optional user-provided `void* ctx` argument to such callbacks, to transfer some state, could you add that?